### PR TITLE
Drone-pypi docs and logo

### PR DESCRIPTION
**Pull request by gjtempleton**

### CODE REVIEW

File: content/gjtempleton/drone-pypi/index.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+image: gjtempleton/drone-pypi
+---
+
+This plugin allows you to publish Python pypi packages as part of your pipeline.
+
+Basic example:
+
+```yaml
+pipeline:
+    pypi_publish:
```
Comment (tonglil) [inline content/gjtempleton/drone-pypi/index.md]: Can you make the yaml indents to 2 spaces please?

Comment (gjtempleton) [reply content/gjtempleton/drone-pypi/index.md]: Sure, done, thanks for the review!

File: content/gjtempleton/drone-pypi/index.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+This block will first generate the relevant .pypirc file with the provided details generating a _repo_ block, then execute the following command:
+
+```
+python3 setup.py sdist upload -r repo
+```
+
+You can also specify the distribution types you want to upload, as well as the repository URL and relative path of the setup python file.
+```diff
+pipeline:
+    pypi_publish:
```
Comment (tonglil) [inline content/gjtempleton/drone-pypi/index.md]: Can you unindent for the rest of the examples as well?
Thanks!

Comment (gjtempleton) [reply content/gjtempleton/drone-pypi/index.md]: Whoops, serves me right for pushing changes in the midst of a meetup. Think that's them all now.

**Pull request merged**